### ISSUE TITLE
Error of lint `#![deny(missing_docs)]` when using `#[rhai::plugin::export_module]`

### ISSUE THREAD

**Issue by Mathieu-Lala** [rhaiscript/rhai#571]
I am using `#[rhai::plugin::export_module]` in my project, and i would like to use the lint `#![deny(missing_docs)]`.
By doing so i have an error of missing documentation :

Reproduce the error in `lib.rs`
```rs
//!

#![deny(missing_docs)]

use rhai::plugin::*;

/// plugin
#[rhai::plugin::export_module] // <-- error here
pub mod my_plugin_module {
    /// double the pride, twice the fall
    pub fn foo(x: i64) -> i64 {
        x * 2
    }
    /// half of the answer
    pub fn bar() -> i64 {
        21
    }
}

#[test]
fn test_module() {
    let mut engine = Engine::new();
    let module = exported_module!(my_plugin_module);
    engine.register_global_module(module.into());

    assert_eq!(engine.eval::<i64>("foo(bar())").unwrap(), 42);
}
```
Producing :
```sh
error: missing documentation for a function
error: missing documentation for a struct
error: missing documentation for an associated function
 --> src/lib.rs:8:1
  |
8 | #[rhai::plugin::export_module]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
```

I believe this can be fixed easily by adding `#[doc(hidden)]` for the generated item

**Comment by schungx** [comment]
Good catch.  The trick is to find out which item is missing documentation -- the macro generates many.

I'll work on that.

**Comment by schungx** [comment]
I have released `1.4.1` of `rhai_codegen` which should resolve this situation.

You can do a `cargo update` to refresh the version to test it.

**Comment by Mathieu-Lala** [comment]
Working well ! Thank you for your speed :scream: !!
